data/json/items/gunmod/muzzle.json

```diff
@@ -70,15 +69,14 @@
     "color": "dark_gray",
     "location": "muzzle",
     "mod_targets": [ "shotgun" ],
-    "damage_modifier": -4,
-    "loudness_modifier": -30,
-    "recoil_modifier": -15
+    "loudness_modifier": -50,
+    "recoil_modifier": -25
```

Comment:
Shouldn't really provide free recoil bonus like that.
Though the loudness modifier could be stronger. Noises of volume 50+ are enough to cover the whole map, it only really matters if it brings it below 30 or so.